search/text_search.cpp

```diff
@@ -12,7 +13,31 @@
 
 /** Main function
  */
+
+std::string lower(std::string word) {  // convert a C++ string to lowercase

```

Comment:
Still missing proper documentation (check the [**typical structure of a program**](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) as a reference).